subsys/bluetooth/mesh/Kconfig

```diff
@@ -628,6 +628,39 @@ config BT_MESH_LABEL_NO_RECOVER
 	  unprovisioned before upgrading to the version with this option). The option is marked as
 	  deprecated to remove the recovery code eventually.
 
+menuconfig BT_MESH_ACCESS_TX_MSG_RANDOMIZER
+	bool "Access layer tx randomizer"
+	help
+	  Enable following of the message transmitting recommendations, the Access layer
+	  specification. The recommendations are optional.
+	  However, they are strictly recommended if device participates in the network with
+	  intensive communication where device receives a lot of requests those require responses.
+
+if BT_MESH_ACCESS_TX_MSG_RANDOMIZER
+
+config BT_MESH_DELAYED_ACCESS_MSG_COUNT
+	int "Number of simultaneously delayed messages"
+	default 4
+	help
+	  The maximum number of delayed access messages the randomizer can manage delaying
+	  at the same time. The number of messages can be handled only if randomizer has
+	  sufficient amount of memory to store model payload for them.
+
+config BT_MESH_DELAYED_ACCESS_MSG_CHUNK_SIZE
+	int "Maximum randomizer memory storage chunk"
+	default 16
+	help
+	  Size of memory that randomizer uses to split model message to. Randomizer allocates
+	  sufficient number of this chunks from the pool to store full model payload.
+
+config BT_MESH_DELAYED_ACCESS_MSG_CHUNK_COUNT
+	int "Maximum number of available chunks"
+	default 20
```

Comment:
We should make it at least so that, at least one model can fit full sized access message.